tests/event-processing-tests/src/main/java/Cleanup.java

```diff
@@ -0,0 +1,54 @@
+import io.cucumber.java.en.And;
```

Comment:
Can we move this to an appropriate package directory, and add the package statement here. I know it’s only test code but best to keep it consistent.